### PR TITLE
Help view

### PR DESCRIPTION
**Pull request by dternyak**
Initial help view - we can build on this later or decide to completely deprecate in favor of knowledge base.

### CODE REVIEW

File: common/components/Header/components/TabsOptions.jsx [hunk 1]
```diff
@@ -23,7 +23,8 @@ const tabs = [
         link: 'view-wallet'
     },
     {
-        name: 'Help'
+        name: 'Help',
```
Comment (dternyak) [inline common/components/Header/components/TabsOptions.jsx]: Probably a better convention than this - maybe react-router even has a way of listing all routes. 

Can fix in a separate PR.

Comment (tayvano) [inline common/components/Header/components/TabsOptions.jsx]: Off the top of my head each 'view' or page or whatever should have a name (Help Me) slug (help-me) which is the url, title (SEO title tag + title shown on page itself), description (OMG wat more SEO?) And ability to hide / show in the nav for different frontends (eg website vs chrome extension) - although this last one should maybe be done on the compile level? 

The ability for me to quickly turn on or off access to a view has come in super handy more than once, and having a single location that controls what views are available for which frontend makes keeping the chrome extension up to date essentially mindless.  

While in the future I hope we don't have 'turn it off and then we can figure it out' moments...it's better to be prepared...especially if I happen to be the only one around.

Comment (tayvano) [inline common/components/Header/components/TabsOptions.jsx]: ps: When I comment like this, take it or leave it. I'll make it clear when I feel strongly about something existing in a particular way. For most things, I'm going to share insightsabout what we've learned and you can use those as you proceed. Please don't make a decision based solely on something I've said, especially if it compromises something else.

Comment (crptm) [inline common/components/Header/components/TabsOptions.jsx]: So re:RR, we should probably switch to v4. No specific reason besides forward compatibility.

**Comment by tayvano** [comment]
I can do the help entries if someone sets the first one up. If we could have separate files for each item, that would work. I am going to keep the help entries much shorter: the most common "Getting Started" and info regarding error messages.

Guides and other questions will be https://myetherwallet.groovehq.com/help_center

Also keep in mind, help is translated and the "save your key" is important to have translated. We used markdown for the current site but I don't think markdown is necessary though, unless we are using it elsewhere already (not another lib = 👍).

**Comment by crptm** [comment]
I think there are only links and bolds from markdown i will prolly ad-hoc parse them.

File: common/components/Header/components/TabsOptions.jsx (hunk 1 above)
Comment (dternyak) [inline common/components/Header/components/TabsOptions.jsx]: @crptm Sorry, I'm a bit confused. What should we switch in v4?

Comment (crptm) [inline common/components/Header/components/TabsOptions.jsx]: @dternyak react-router, we are on 3.x release latest is 4.x

**Comment by dternyak** [comment]
@crptm Yeah, separate PR to start laying out a way for @tayvano to be able to setup help entries.

**Pull request merged**